core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -580,7 +581,7 @@ public boolean isSupported(final Class<? extends Operation> operationClass) {
      * @return a collection of all the supported {@link Operation}s.
      */
     public Set<Class<? extends Operation>> getSupportedOperations() {
-        return operationHandlers.keySet();
+        return new HashSet<>(operationHandlers.keySet());
```

Comment:
Why is this needed? It seems unrelated to this PR.

Reply:
I wanted to safely return a copy of the keys so I could modify the result without modifying the underlying operationHandlers map